clients/venice-push-job/src/test/java/com/linkedin/venice/hadoop/TestVenicePushJobConfig.java

```diff
@@ -119,14 +119,12 @@ private ControllerClient getClient(Consumer<StoreInfo> storeInfo) {
 
   private StoreInfo getStoreInfo(Consumer<StoreInfo> info) {
     StoreInfo storeInfo = new StoreInfo();
-    storeInfo.setIncrementalPushEnabled(false);
```

Comment:
Did we remove this field/API from StoreInfo class as well? I don't see it in this PR.

Reply:
we'll remove it later